### PR TITLE
[New Entry] General: Virtual Reality

### LINKED ISSUE #1010: [Entry] General: Virtual reality
### Reason/inspiration (optional)

Currently there is an entry for AR: https://www.codecademy.com/resources/docs/general/augmented-reality

There's a section that mentions AR vs VR but it would be beneficial to create a separate VR entry. Edits to the AR entry may also be needed.

### Entry Type (select all that apply)

New Concept Entry for Existing Topic

### Code of Conduct

- [X] I agree to follow Codecademy Doc's Code of Conduct

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by yesimhozman**
### Description

In the contents, the AR and VR topics are shown as separate entries now. VR page has Title, description, Subjects, Tags, CatalogContent , Augmented Reality vs. Virtual Reality section,  and Virtual Reality Examples. 
I fix a typo on the AR page too. 

Closes #1010 

### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/general/concepts/virtual-reality/virtual-reality.md
Comment (SSwiniarski) [inline content/general/concepts/virtual-reality/virtual-reality.md:16]:
```suggestion
Virtual reality (VR) is a technology that uses head-mounted headsets with displays to generate realistic images, sounds, and other sensations to put users into an immersive virtual environment. VR can create unbounded worlds that people can walk around and interact with using their hands, to feel as if they were transported to another place.
```

Preference is for Docs to avoid direct address to the reader in the form of using "you", "we", or "us."

File: content/general/concepts/virtual-reality/virtual-reality.md
Comment (SSwiniarski) [inline content/general/concepts/virtual-reality/virtual-reality.md:22]:
```suggestion
The difference between augmented reality (AR) and virtual reality (VR) is that augmented reality utilizes the current physical setting of the user while introducing a layer of synthetic digital, sonic or somatic presentation to enhance the user experience whereas VR involves use of a headset to facilitate an experience rooted in the graphic and sonic presentation within the VR setup as opposed to using the individual's more broad physical surroundings for interaction.
```

File: content/general/concepts/virtual-reality/virtual-reality.md
Comment (SSwiniarski) [inline content/general/concepts/virtual-reality/virtual-reality.md:31]:
```suggestion
- Real estate:  explore houses online and save time.
```

File: content/general/concepts/virtual-reality/virtual-reality.md
Comment (SSwiniarski) [inline content/general/concepts/virtual-reality/virtual-reality.md:24]:
```suggestion
Virtual reality is poised to grow substantially in use during the 21st century as it has many potential uses that range from entertainment, medical, commercial, military, fashion and design etc.
```

File: content/general/concepts/virtual-reality/virtual-reality.md
Comment (SSwiniarski) [inline content/general/concepts/virtual-reality/virtual-reality.md:18]:
```suggestion
It’s still considered the wild west of the tech industry⁠— fun, exciting, bustling of opportunities and things are looking very promising.
```

File: content/general/concepts/virtual-reality/virtual-reality.md
Comment (SSwiniarski) [inline content/general/concepts/virtual-reality/virtual-reality.md:28]: I'd like to see more specific examples like this that could be linked out to.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@yesimhozman, I did a first review and left a couple of comments.

**Comment by yesimhozman** [comment]
> @yesimhozman, I did a first review and left a couple of comments.

Thanks for the comments. I made changes and added links for examples

**Review by SSwiniarski — APPROVED** [review]
@yesimhozman, Looks good for a second review.

**Review by HishamT — APPROVED** [review]
LGTM

File: content/general/concepts/augmented-reality/augmented-reality.md
Comment (yangc95) [inline content/general/concepts/augmented-reality/augmented-reality.md:22]: Should be also link VR here?

File: content/general/concepts/virtual-reality/virtual-reality.md
Comment (yangc95) [inline content/general/concepts/virtual-reality/virtual-reality.md:18]:
```suggestion
It’s still considered the wild west of the tech industry⁠ — fun, exciting, bustling of opportunities and things are looking very promising.
```

File: content/general/concepts/virtual-reality/virtual-reality.md
Comment (yangc95) [inline content/general/concepts/virtual-reality/virtual-reality.md:16]:
```suggestion
Virtual reality (VR) is a technology that uses head-mounted headsets with displays to generate realistic images, sounds, and other sensations to put users into an immersive virtual environment. VR can create unbounded worlds that people can walk around and interact with using their hands, and to feel as if they were transported to another place.
```

**Comment by yangc95** [comment]
@yesimhozman This looks almost merge-ready! I just had some last-minute comments. In addition, make sure to update your branch and run scripts as needed to pass the checks. If you need any assistance with any of these, please let me know and I can help out right before the merge. Thanks!

File: content/general/concepts/augmented-reality/augmented-reality.md
Comment (HishamT) [inline content/general/concepts/augmented-reality/augmented-reality.md:22]: This sentence is quite long! Any chance it could be broken up?

**Review by HishamT — CHANGES_REQUESTED** [review]
@yesimhozman, I have finished reviewing your entry and left a few comments.

File: content/general/concepts/augmented-reality/augmented-reality.md
Comment (yesimhozman) [reply content/general/concepts/augmented-reality/augmented-reality.md:22]: Absolutely, thanks for review @yangc95

Comment (yesimhozman) [reply content/general/concepts/augmented-reality/augmented-reality.md]: Hi @HishamT . I break this sentence into two.

**Review by HishamT — APPROVED** [review]
LGTM!

File: content/general/concepts/virtual-reality/virtual-reality.md
Comment (Dusch4593) [inline content/general/concepts/virtual-reality/virtual-reality.md:35]:
```suggestion
- Web development: [A-Frame](https://www.codecademy.com/learn/learn-a-frame) is a popular [open-source](https://www.codecademy.com/resources/docs/open-source) web framework for building virtual reality experiences in the web browser.
- Automotive industry: Companies like [Jaguar](https://techwireasia.com/2020/02/how-jaguar-land-rover-uses-vr-to-design-and-engineer-new-vehicles/) are incorporating VR into their vehicle design before commissioning expensive prototypes.
- Healthcare:
  - Softwares like [Osso VR](https://www.ossovr.com/) allow for practicing surgical techniques on virtual bodies.
  - [BehaVR](https://www.behavr.com/) is used as pain relief for burn injuries.
- Real estate: [Klapty](https://www.klapty.com/explore) features virtual tours of apartments and houses that are on the market.
- Education: Products like [Engagevr](https://engagevr.io) provides learning experiences in a risk-free space mostly for soft skills such as public speaking and active listening.
- Tourism: [Globetrotter](https://globetrotter-vr.com/) features interactive virtual tours of places around the world for users to consider which activities and destinations to try out.
```

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@yesimhozman Finished my review and left some comments/suggestions. Could we also make the following change to line # 16 in the `augmented-reality.md` file?

> \*\*Augmented reality (AR)\*\* is a digitally enhanced real world experience provided through hardware. In many instances, AR provides an additional sensory experience to natural settings. A popular example of AR in mobile applications is the video game Pokémon Go.

**Comment by yesimhozman** [comment]
> 

hi @Dusch4593 , thanks for the review. Finished my last commits

**Review by Dusch4593 — APPROVED** [review]
@yesimhozman LGTM! 👍🏻

**Comment by Dusch4593** [comment]
Congrats and thank you so much for contributing to Codecademy Docs, @yesimhozman!! 🚀 🙌🏻 🥳

Your edits in General: Augmented Reality should now be live: https://www.codecademy.com/resources/docs/general/augmented-reality

Same with the new entry: https://www.codecademy.com/resources/docs/general/virtual-reality

Currently, your Codecademy profile is listed anonymously on these entries. To received credit and be added to the page, you'll need to add your GitHub username to your Codecademy profile, log out of Codecademy, and log back in through your GitHub account. Let us know if you'd like assistance with doing this.

Screen Shot 2022-10-31 at 5 20 59 PM

Looking forward to your next contribution to Docs! 😄

**Comment by yesimhozman** [comment]
thank you so much @Dusch4593 .  It was one of my first open-source contributions 🥳  I log out and log in with my github account. But, it is still an anonymous contributor.  What should I do?

**Comment by yangc95** [comment]
@yesimhozman Is your GitHub username updated on your Codecademy profile as well? When you first log onto Codecademy and are brought to your dashboard, if you go to the upper right 🥪 button -> click your name at the bottom -> Profile -> Edit Profile button -> and add your username there + make sure your profile (not name) is public + save, that should do the trick.

If the steps above are already done, please send a screenshot of what your profile currently looks like (with any sensitive info censored).

**Comment by yesimhozman** [comment]
imageimage . Hello thanks for detailed explanation, it looks like that @yangc95

**Comment by yangc95** [comment]
Great! Looks like you're showing up as the contributor. I recommend changing your profile pic if you want something more customized. Other than that, looks like you're set!

Screen Shot 2022-11-01 at 6 29 53 PM

**Pull request merged**
